schema/Makefile

```diff
@@ -150,23 +148,23 @@ install_pgtap: pgtap
 
 .PHONY: install_cpanm
 install_cpanm:
-ifeq (${shell which ${CPANM}},)
+ifeq ($(shell which $(word 1,$(CPANM))),)
 	# install cpanm
-	@@echo | ${CPAN} # accept cpan defaults blindly
-	@@${CPAN} App:cpanminus
+	@@echo | $(CPAN) # accept cpan defaults blindly
+	@@$(CPAN) App:cpanminus
 endif
 
 .PHONY: install_cpandeps
 install_cpandeps:
 	# install sqitch
-	${CPANM} -n https://github.com/matthieu-foucault/sqitch/releases/download/v1.0.1.TRIAL/App-Sqitch-v1.0.1-TRIAL.tar.gz
+	$(CPANM) -n https://github.com/matthieu-foucault/sqitch/releases/download/v1.0.1.TRIAL/App-Sqitch-v1.0.1-TRIAL.tar.gz
```

Comment:
are we still waiting for a release on this? If so, maybe we'll migrate to a bcgov repo

Reply:
My PR there has some comments that need to be taken care of in order for it to be merged. We're not using the feature that was developed anyway, so we can revert to sqitch 0.9999.